Kickstarter-iOS/AppEnvironmentType.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import Library
+
+public protocol AppEnvironmentType {
+  static func logout()
+}
+
+extension AppEnvironment: AppEnvironmentType {
```

Comment:
What would you say about moving this to the `AppEnvironment` file instead? It will be more clear if the struct itself is defined as conforming to this protocol, instead of hiding it here in the protocol file?

So if we do this we can remove the `import Library` here.

and move the conformance to

```
// AppEnvironment.swift
public struct AppEnvironment: AppEnvironmentType { .. }
```

This will then follow the same conventions as `Service` & `ServiceType`